src/Controls/src/Core/RefreshView/RefreshView.cs

```diff
@@ -135,6 +112,20 @@ public IPlatformElementConfiguration<T, RefreshView> On<T>() where T : IConfigPl
 			return _platformConfigurationRegistry.Value.On<T>();
 		}
 
+		ICommand ICommandElement.Command => Command;
+
+		object ICommandElement.CommandParameter => CommandParameter;
+
+		protected override bool IsEnabledCore => base.IsEnabledCore && CommandElement.GetCanExecute(this);
+		
+		void ICommandElement.CanExecuteChanged(object sender, EventArgs e)
+		{
+			if((bool)GetValue(IsRefreshingProperty))
+				return;
```

Comment:
Hi, @mattleibow I've put it here because I wanted to replicate an already existing behavior. I'm also not sure why it was there.
<img width="648" alt="Screenshot 2024-09-03 at 17 20 40" src="https://github.com/user-attachments/assets/978e3082-c377-4609-bdd7-090fb03944fe">

Reply:
OK, makes sense for now. If it becomes a problem, then we can say we didn't touch it!

Reply:
Yea, that was my way of thinking 😅